src/test/kfs_rest_test.cpp

```diff
@@ -197,3 +199,38 @@ TEST_F(HttpRestApiHandlerTest, dispatchReady) {
 
     ASSERT_EQ(c, 1);
 }
+
+TEST_F(HttpRestApiHandlerTest, serverReady) {
+
+    ovms::HttpRequestComponents comp;
+    comp.type = ovms::KFS_GetServerReady;
+    std::string request;
+    std::string response;
+    ovms::Status status = handler->dispatchToProcessor(request, &response, comp);
+
+    ASSERT_EQ(status, server->isReady() ? ovms::StatusCode::OK : ovms::StatusCode::MODEL_NOT_LOADED);
+}
+
+TEST_F(HttpRestApiHandlerTest, serverLive) {
+
+    ovms::HttpRequestComponents comp;
+    comp.type = ovms::KFS_GetServerLive;
+    std::string request;
+    std::string response;
+    ovms::Status status = handler->dispatchToProcessor(request, &response, comp);
+
+    ASSERT_EQ(status, server->isLive() ? ovms::StatusCode::OK : ovms::StatusCode::INTERNAL_ERROR);
+}
```

Comment:
This test does not ensure that when the Server is Live the response is correct as it does not perform setup to ensure that the server is actually up.

You can either mock-up the server to verify the dependency of response from server state or write test/modify the grpc one to check the liveness response. With mock you only unit-test, with full integration test you test whole server behavior so I think the second approach is better. Note that REST liveness should respond on HTTP server state while readiness on Servable Manager module.